ao.php

```diff
@@ -491,6 +491,44 @@ function ao_civicrm_alterAPIPermissions($entity, $action, &$params, &$permission
   }
 }
 
+/**
+ * Implements hook_civicrm_pre().
+ *
+ * AOS-64 Ensure that all postcodes are formatted in the format of xxx yyy
```

Comment:
This function seems to be oriented towards fixing up the spaces in postal code so that fourth character is a space (when length of string after spaces removed is >3). That is useful. Is there a validation elsewhere that a#a #a# is the regex pattern?

Reply:
@JoeMurray none as yet as far as I know